pandas/tests/extension/base/methods.py

```diff
@@ -127,10 +127,11 @@ def test_combine_add(self, data_repeated):
         s1 = pd.Series(orig_data1)
         s2 = pd.Series(orig_data2)
         result = s1.combine(s2, lambda x1, x2: x1 + x2)
-        expected = pd.Series(
-            orig_data1._from_sequence([a + b for (a, b) in
-                                       zip(list(orig_data1),
-                                           list(orig_data2))]))
+        with np.errstate(over='ignore'):
```

Comment:
what surfaces the error here?

Reply:
When testing with Int8 dtype, you get an overflow warning for summing the scalars (the `a + b` below)

Reply:
hmm ok, maybe we should actually move this to the integer tests rather than here then?

Reply:
I think it is fine to ignore it here (it's numpy behaviour to warn, not something we need to test), but it's true that we might want to add explicit test for the behaviour we want on the Series level in the integer tests. 

The behaviour currently to silently ignore it in the case of series is consistent with a Series with numpy dtype int8:

```
In [38]: s = pd.Series([50, 100], dtype='Int8')

In [39]: s[0] + s[1]
/home/joris/miniconda3/envs/dev/bin/ipython:1: RuntimeWarning: overflow encountered in byte_scalars
  #!/home/joris/miniconda3/envs/dev/bin/python
Out[39]: -106

In [40]: s + s[::-1].reset_index(drop=True)
Out[40]: 
0    -106
1    -106
dtype: Int8
```

Reply:
i understand though this may be indicative of an error in other extension types. ok for now